boards/posix/native_posix/cmdline.c

```diff
@@ -66,6 +81,13 @@ void native_handle_cmd_line(int argc, char *argv[])
 		"97229 (decimal), 0x17BCD (hex), or 0275715 (octal)"},
 #endif
 
+#if defined(CONFIG_BT_USERCHAN)
+		{ false, false, false,
```

Comment:
If the parameter is mandatory, maybe better set mandatory to true (the 2nd parameter in this line), so the help message reflects it.

Reply:
The way I was thinking, is that the errors would only come once the application decides it wants to use Bluetooth, i.e. calls the bt_enable() API (which will return an error, which all the Bluetooth-using samples will print to console). I'm not convinced that this is the best approach though, i.e. the assumption that there may apps that enable Bluetooth in Kconfig but never call bt_enable().

Reply:
That mandatory switch only affects the syntax displayed in the help message. So you can still throw the error at any point later (or never).
If you want the error displayed ASAP, you could print it after the for loop in `native_handle_cmd_line()` if the option was not found.
If you want to postpone it to the driver init you can also do that.
Anyhow, enabling bluetooth but never using it would seem more like a misconfiguration.